### PR TITLE
adding data integration (loading data from pretalx with a 1min ttl)

### PR DESCRIPTION
**Pull request by martinheidegger**
This PR adds basic data integration from pretalx to the netlify site generator. It loads all session and user data (including images & resources).

As the schedule is using the https://conf.consento.org/template-test schedule data for the visualization which has some tricks in there that should work.

For the test, the data is still `.gitignore`ed to make sure that the git repository is not spammed with commits - so you need to run `npm run build` in the root directory after `npm install`

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "private": "true",
+  "scripts": {
+    "build": "netlify build",
+    "postinstall": "cd website; echo 'installing website dependencies'; npm i; echo 'installing plugin dependencies'; cd plugins/pretalx-fetch; npm i"
```
Comment (ssaavedra) [inline package.json]: Why not leave the postinstall for the plugin dependency on website/package.json?

**Comment by ssaavedra** [comment]
I cannot run `netlify build` or `npm i` on the root folder because I'm not the Netlify account holder :-(

**Comment by martinheidegger** [comment]
@ssaavedra I updated the build scripts to run smoother and without netlify (you should be able to just run `npm run dev` in the dev folder) - netlify deploy example: https://deploy-preview-25--dat-events.netlify.app/2020/schedule/

**Comment by martinheidegger** [comment]
I looked a bit into calendar visualization. [fullcalendar](https://fullcalendar.io/) is the visualization that pretalx uses but in tui ([link to docs](https://github.com/nhn/tui.calendar/blob/master/docs/getting-started.md#support-timezone)) it should be easily possible to add multiple timezones.

**Comment by martinheidegger** [comment]
I think the schedule display should work as-is, now the speaker/talk/speakers/talks displays need some love.

**Comment by Frando** [comment]
I was finally able to properly test this out - it looks great! Thanks a lot martin for all the work done here. From initial review this looks great to be merged and we can still improve in the process.

**Comment by Frando** [comment]
I see there is some code to change the timezone in `calender.js` but it seems it's not yet active - is this an unfinished part or broken? I can look into it today.

File: website/content/2020/schedule.11ty.js [hunk 2]
```diff
@@ -55,7 +55,7 @@ function list (iterable, mapper, sep='', lastSep) {
   return `${entries.slice(0, entries.length - 1).join(sep)}${lastSep}${entries[entries.length-1]}`
 }
 
-module.exports = class Schedule {
+class Schedule {
```
Comment (martinheidegger) [inline website/content/2020/schedule.11ty.js]: Does this work?

Comment (ssaavedra) [reply website/content/2020/schedule.11ty.js]: No, it shouldn't have cleanly merged. This was a mistake.

**Comment by dpaez** [comment]
Looks great! <3

**Pull request merged**